department-of-veterans-affairs/abd-vro
# Move Lighthouse API Mock Svc

## What was the problem?

Mock for the lighthouse api svc lingering around

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

moves it to the repo, hosting the deprecated svc-lighthouse-api https://github.com/department-of-veterans-affairs/abd-vro-svc-lighthouse-api/pull/4

## How to test this PR
- No build issues locally


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
